### ISSUE TITLE
Volume issues with (bassy) music

### ISSUE THREAD

**Issue by BennoMP** [AlanFinotty1995/AAF_DCH_RealtekModded#22]
Hey all,
If I play music such as Culprate - Aurora, I get random volume changes at bassy parts.
I have Sound Blaster 720 installed and I am using the latest driver (Realtek HDA control panel, not the app). Sound chip in question is the Realtek ALC261.
The driver policy for effects has been set correctly.
All settings are default, except for the multiple sound devices option and quality (24bit96000hz):
![image](https://user-images.githubusercontent.com/61586142/92303366-5df7ff00-ef74-11ea-8728-61e70dcd36f5.png)

For the Sound Blaster 720 software I have the Concert preset enabled in it's default settings.

I already checked if it was the communications setting in Windows but that was set correctly:
![image](https://user-images.githubusercontent.com/61586142/92303406-c3e48680-ef74-11ea-8cd7-c010ab631ef4.png)

PS: The Dolby stuff in HDA is also disabled

**Comment by RCCore** [comment]
@BennoMP what motherboard do you have?

**Comment by BennoMP** [comment]
HP 3647h. Does that matter?

**Comment by BennoMP** [comment]
![92303366-5df7ff00-ef74-11ea-8728-61e70dcd36f5](https://user-images.githubusercontent.com/61586142/92306695-6c084880-ef91-11ea-9184-62400d48543a.png)
I might have found something. This was enabled on the stock drivers and the AAF Optimus (non-DCH). I enabled it now (this is the old screenshot).
I'm going to do some further testing and report my results here, but so far I haven't had any issues with songs I used to test.
